rq/utils.py

```diff
@@ -297,3 +297,21 @@ def split_list(a_list, segment_size):
     """
     for i in range(0, len(a_list), segment_size):
         yield a_list[i:i + segment_size]
+
+
+def get_call_string(func_name, args, kwargs):
+    """Returns a string representation of the call, formatted as a regular
+    Python function invocation statement.
+    """
+    if func_name is None:
+        return None
+    from .job import truncate_long_string  # work around circular import issue
+
+    arg_list = [as_text(truncate_long_string(repr(arg))) for arg in args]
+
+    kwargs = ['{0}={1}'.format(k, as_text(truncate_long_string(repr(v)))) for k, v in kwargs.items()]
+    # Sort here because python 3.3 & 3.4 makes different call_string
+    arg_list += sorted(kwargs)
```

Comment:
We can drop the sort here since we don't need to worry about Python 3.3 or 3.4 anymore.

Reply:
Good catch. The comment was outdated so I removed it.

Anyway, I would keep the sort but for a different reason.

Since rq supports Python ≥ 3.5 and dict ordering is guaranteed starting with Python 3.7, I'd sort to keep function output deterministic. Otherwise, for the same arguments, the result might differ in different Python implementations/versions.

What do you think?

Reply:
Ok.